### ISSUE TITLE
addModuleImportToModule doesn't work in 2.0.1, works fine in 2.0.0

### ISSUE THREAD

**Issue by mraible** [nitayneeman/schematics-utilities#22]
See builds in [this PR](https://github.com/oktadeveloper/schematics/pull/204#commits-pushed-304d41b) with an upgrade to 2.0.1 (fails) and subsequent downgrade to 2.0.0 (passes).

**Comment by mraible** [comment]
I'm not sure 2.0.0 is working correctly either. It seems to pull in the latest version of a dependency, rather than the version I specify. I have:

```
const dependencies: NodeDependency[] = [];

dependencies.push({type: NodeDependencyType.Default, version: '1.2.2', name: '@okta/okta-angular'})

dependencies.forEach(dependency => {
  addPackageJsonDependency(host, dependency);
  context.logger.log('info', `✅️ Added '${dependency.name}' into ${dependency.type}`);
});
```

However, v1.3.0 is installed, rather than 1.2.2.

```
npm WARN @okta/okta-angular@1.3.0 requires a peer of @angular/common@>=4 
  but none is installed. You must install peer dependencies yourself.
```

**Comment by nitayneeman** [comment]
Closing this issue - should be solved with v2.0.2.
